Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -603,6 +604,91 @@ public protocol AppBackend: Sendable {
     /// ``showAlert(_:window:responseHandler:)``.
     func dismissAlert(_ alert: Alert, window: Window?)
 
+    /// Creates a sheet object (without showing it yet). Sheets contain View Content.
+    /// They optionally execute provied code on dismiss and
+    /// prevent users from interacting with the parent window until dimissed.
+    func createSheet() -> Sheet
+
+    /// Updates the content and appearance of a sheet
+    func updateSheet(
+        _ sheet: Sheet,
+        content: Widget,
+        onDismiss: @escaping () -> Void
+    )
+
+    /// Shows a sheet as a modal on top of or within the given window.
+    /// Users should be unable to interact with the parent window until the
+    /// sheet gets dismissed. The sheet will be closed once onDismiss gets called
+    ///
+    /// Must only get called once for any given sheet.
+    ///
+    /// If `window` is `nil`, the backend can either make the sheet a whole
+    /// app modal, a standalone window, or a modal for a window of its choosing.
+    func showSheet(
+        _ sheet: Sheet,
+        window: Window?
+    )
+
+    /// Dismisses a sheet programmatically.
+    /// Gets used by the SCUI sheet implementation to close a sheet.
+    func dismissSheet(_ sheet: Sheet, window: Window?)
+
+    /// Sets the corner radius for a sheet presentation.
+    ///
+    /// This method is called when the sheet content has a `presentationCornerRadius` modifier
```

Comment:
Doesn't have to be at the top level, just has to have the preference key set. (same for the other presentation methods down below)